pkg/oc/cli/admin/ipfailover/ipfailover.go

```diff
@@ -60,189 +57,245 @@ var (
 	  %[1]s %[2]s ipf-alt --selector="hagroup=us-west-ha" --virtual-ips="1.2.3.4" -o yaml --images=myrepo/myipfailover:mytag`)
 )
 
-func NewCmdIPFailoverConfig(f kcmdutil.Factory, parentName, name string, streams genericclioptions.IOStreams) *cobra.Command {
-	options := &ipfailover.IPFailoverConfigCmdOptions{
-		Action: configcmd.BulkAction{
-			IOStreams: streams,
-		},
-		ImageTemplate:    variable.NewDefaultImageTemplate(),
-		ServiceAccount:   "ipfailover",
-		Selector:         ipfailover.DefaultSelector,
-		ServicePort:      ipfailover.DefaultServicePort,
-		WatchPort:        ipfailover.DefaultWatchPort,
-		NetworkInterface: ipfailover.DefaultInterface,
-		VRRPIDOffset:     0,
-		Replicas:         1,
+type IPFailoverConfigCmdOptions struct {
```

Comment:
`IPFailoverConfigOptions` - we don't use `Cmd` anywhere, we should not introduce it here. Honestly, since the commands name is ipfailover the struct name should be just `IPFailoverOptions`.

Reply:
+1 on `IPFailoverOptions`